src/main/java/org/opensearch/ad/ml/ModelManager.java

```diff
@@ -469,8 +469,7 @@ private void trainModelForStep(
             .compact(true)
             .precision(Precision.FLOAT_32)
             .boundingBoxCacheFraction(AnomalyDetectorSettings.REAL_TIME_BOUNDING_BOX_CACHE_RATIO)
-            // for external shingling, rcf does not recognize shingle
-            .shingleSize(1)
+            .shingleSize(detector.getShingleSize())
```

Comment:
Will this impact the model memory calculation?

Reply:
Not much, in my estimate ...